AZ3166/src/cores/arduino/EEPROMInterface.h

```diff
@@ -13,13 +13,13 @@
 
 #define WIFI_SSID_MAX_LEN       32
 #define WIFI_PWD_MAX_LEN        64
-#define AZ_IOT_HUB_MAX_LEN      200
+#define AZ_IOT_HUB_MAX_LEN      512
```

Comment:
200 is not enough?
512 will cause more memory consumption

Reply:
Looking at EEPROMInterface.cpp
```
// The segment length of each data partition
const static int DATA_SEGMENT_LENGTH[11] = {976, 0, 192, 120, 0, 584, 680, 784, 880, 0, 88};
```

We already have `584` blocked for that.

Besides, device id etc. are not user defined and hold together with scope id and key. So `200` wasn't enough for the sample auth info I had.

Reply:
512 will slightly increase the memory consumption, and especially it's stack